src/backend/app/central/central_routes.py

```diff
@@ -54,3 +56,35 @@ async def get_form_lists(
     """
     forms = await central_crud.get_form_list(db)
     return forms
+
+
+@router.post("/refresh_appuser_token/{project_id}")
+async def refresh_appuser_token(
+    project_id: int,
+    current_user: db_models.DbUser = Depends(project_admin),
+    db: Session = Depends(database.get_db),
+):
+    """Refreshes the token for the app user associated with a specific project.
+
+    Args:
+        project_id (int): The ID of the project to refresh the app user token for.
+        current_user: The current authenticated user with project admin privileges.
+        db: The database session to use.
+
+    Returns:
+        The refreshed app user token.
+    """
+    odk_credentials = await project_deps.get_odk_credentials(db, project_id)
+    project = await project_deps.get_project_by_id(db, project_id)
+    project_odk_id = project.odkid
+    db_xform = await project_deps.get_project_xform(db, project_id)
+    odk_token = await central_crud.get_appuser_token(
+        db_xform.odk_form_id, project_odk_id, odk_credentials, db
+    )
+    if odk_token:
+        project.odk_token = odk_token
+    db.commit()
+    return {
+        "status_code": 200,
```

Comment:
There id an enum available that would be good to use here:

200 --> HTTPStatus.OK (or maybe SUCCESS instead, I forget now)